### PR TITLE
link fix

### PR DESCRIPTION
**Pull request by RichardSmedley**

### CODE REVIEW

File: modules/learn/pages/data/data.adoc [hunk 1]
```diff
@@ -132,7 +132,7 @@ xref:clusters-and-availability/xdcr-conflict-resolution.adoc[XDCR Conflict Resol
 If non-zero, this determines the point at which the document is removed from the system.
 The value can be set either per-document, by means of Couchbase SDK APIs (which is referred to as _TTL_); or per bucket, by means of Couchbase Web Console, the Couchbase CLI, or the Couchbase REST API (which is referred to as _Bucket TTL_).
 +
-For information on _TTL_, see xref:java-sdk::core-operations.adoc[Core Operations].
+For information on _TTL_, see xref:java-sdk:howtos:kv-operations.adoc#expiration-ttl[Core Operations].
```
Comment (eric-schneider) [inline modules/learn/pages/data/data.adoc]:
```suggestion
For information on _TTL_, see xref:java-sdk:howtos:kv-operations.adoc[Key Value Operations].
```

I don't see the `#expiration-ttl` ID on the target page. Is this ID on an included partial, or perhaps you're planning to add it to the page directly in a parallel PR? The suggestion above is a potential alternative if you don't have a direct target for this at the moment.

**Review by eric-schneider — CHANGES_REQUESTED** [review]

File: modules/learn/pages/data/data.adoc (hunk 1 above)
Comment (RichardSmedley) [reply modules/learn/pages/data/data.adoc]: Thanks - will fix with Java's version of the anchor tag...

File: modules/learn/pages/data/data.adoc [hunk 2]
```diff
@@ -132,7 +132,7 @@ xref:clusters-and-availability/xdcr-conflict-resolution.adoc[XDCR Conflict Resol
 If non-zero, this determines the point at which the document is removed from the system.
 The value can be set either per-document, by means of Couchbase SDK APIs (which is referred to as _TTL_); or per bucket, by means of Couchbase Web Console, the Couchbase CLI, or the Couchbase REST API (which is referred to as _Bucket TTL_).
 +
-For information on _TTL_, see xref:java-sdk::core-operations.adoc[Core Operations].
+For information on _Expiration_, see xref:java-sdk:howtos:kv-operations.adoc#document-expiration[KV Operations].
```
Comment (eric-schneider) [inline modules/learn/pages/data/data.adoc:135]:
```suggestion
For information on _Expiration_, see xref:java-sdk:howtos:kv-operations.adoc#document-expiration[Key Value Operations].
```

**Review by eric-schneider — APPROVED** [review]
lgtm, though I offered a suggestion to change the xref link text to be the full title of the target page.

**Pull request merged**